src/components/CollectionBox.css

```diff
@@ -18,4 +18,3 @@
     padding: 10px;
     width: 100%;
   }
-  
```

Comment:
Probably being picky, but can this extra line be removed?

Reply:
To clarify, are you asking if something will break if it is removed?

Reply:
I just meant we can probably remove it. I doubt it would break anything since the pre-checks passed.

Reply:
Oh, the red means that the extra line was removed in my commit! So it was there before and now it isn't.